arch/arm/src/efm32/efm32_clockconfig.c

```diff
@@ -718,7 +689,7 @@ static inline uint32_t efm32_lfaclk_config(uint32_t lfaclksel, bool ulfrco,
 static inline uint32_t efm32_lfbclk_config(uint32_t lfbclksel, bool ulfrco,
                                            uint32_t hfcoreclk)
 {
-  uint32_t lfbclk;
+  uint32_t lfbclk = 0;
```

Comment:
Maybe better to add assignment to LFRCO case below? Is `BOARD_LFRCO_FREQUENCY` not defined?

Reply:
Done.